src/Microsoft.OData.Client/DataServiceContext.cs

```diff
@@ -1207,11 +1207,11 @@ public virtual QueryOperationResponse LoadProperty(object entity, string propert
         }
 
         /// <summary>Loads a page of related entities by using the supplied next link URI.</summary>
-        /// <returns>An instance of <see cref="T:Microsoft.OData.Client.QueryOperationResponse`1" /> that contains the results of the request.</returns>
+        /// <returns>An instance of <see cref="Microsoft.OData.Client.QueryOperationResponse`1" /> that contains the results of the request.</returns>

```

Comment:
I didn't resolve ALL warnings. I will split this across multiple pull requests, as otherwise it will get too big to review. There will still be warnings left after this pull request is completed. Either way, your comment is definitely valid, I added the change.